computation/src/main/java/com/powsybl/computation/ExecutionReport.java

```diff
@@ -1,42 +1,31 @@
 /**
- * Copyright (c) 2016, All partners of the iTesla project (http://www.itesla-project.eu/consortium)
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
```

Comment:
You cannot remove the old copyright.

It seems there is an issue with a rename / move of a file.